tes-adapter/scripts/run_get_tes-adapter_jar.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+workdir=$(pwd);
+
+if [ -n "$2" ]; then
+    gitRepoUrl="-b $1 $2"
+else
+    gitRepoUrl=$1
+fi
+
+mkdir tmp;
+cd tmp;
+
+tmpdir=$(pwd);
+
+git clone $gitRepoUrl $tmpdir;
+
+./gradlew bootJar;
+echo "Copying jar-file from tes-adapter to $workdir"
```

Comment:
also log the other stage of the script like cloning, building

Reply:
added